src/bun.js/api/bun/dns_resolver.zig

```diff
@@ -1962,6 +1962,9 @@ pub const DNSResolver = struct {
 
     pub fn onDNSPollUv(watcher: [*c]bun.windows.libuv.uv_poll_t, status: c_int, events: c_int) callconv(.C) void {
         const poll = UvDnsPoll.fromPoll(watcher);
+        const vm = poll.parent.vm;
+        vm.eventLoop().enter();
```

Comment:
Without this enter/exit callback like this wouldn't be called. I thought this would fix the dns tests, but it did not.
```js
const dns = require("dns");
dns.resolveNs("", (err, results) => {
  console.log(results);
});
```